src/PaymentPart/Output/FpdfOutput/FpdfTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput;
+
+trait FpdfTrait
+{
+    /**
+     * Set dash line style for the next ->Line() command
+     * DO NOT USE IT. Its purpose is only for the Swiss QR Bill library. Name and/or
+     * signature may change without any notification.
+     *
+     * @see http://www.fpdf.org/en/script/script33.php
+     * @param float $black Width of black dash. 0 to reset to solid line
+     * @param ?float $white Width of white dash. Defaults to same as black
+     * @return void
+     */
+    public function swissQrBillSetDash(float $black = 0, ?float $white = null): void
+    {
+        $white = $white === null ? $black : $white;
+
+        $s = $black > 0
+            ? sprintf('[%.3F %.3F] 0 d', $black * $this->k, $white * $this->k)
+            : '[] 0 d';
+
+        $this->_out($s);
+    }
+
+    /**
+     * Rotate text
+     * DO NOT USE IT. Its purpose is only for the Swiss QR Bill library. Name and/or
+     * signature may change without any notification.
+     *
+     * @see http://www.fpdf.org/en/script/script31.php
+     * @param float $x
+     * @param float $y
+     * @param string $txt
+     * @param float $txtAngle
+     * @param float $fontAngle
+     * @return void
```

Comment:
IMO these can be remove as they add no additional information to whats available in code.